cli/cmd/usergroup/add.go

```diff
@@ -34,11 +37,21 @@ func AddCmd(ch *cmdutil.Helper) *cobra.Command {
 			}
 
 			if projectName != "" {
+				resources, err := cmdutil.ParseResourceStrings(explores, canvases)
```

Comment:
In the case where `projectName == ""`, it should error if `restrictResources` or `explores` or `canvases` are non-empty